### DISCUSSION TITLE
Access Issue Due to Lost Phone and 2FA

### DISCUSSION THREAD

**Discussion by baharbaba** [community/community#137704] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I’m unable to access my account due to a lost phone and the Microsoft Authenticator app. I’ve previously submitted ID verification for GitHub Student status and can resubmit if necessary. Additionally, I can provide proof of ownership for the website listed on my profile. Please advise on the steps to regain access.

Edit: i also have my discord account connected to my github

**Comment by davevad93** [comment, marked as answer]
Hi @ozturkyusufziya ,

The Community is unable to help with account related issues, the best way to find a solution to your problem is to contact support, open a ticket [here](https://support.github.com/contact/account), select `Two factor authentication recovery` or `Device verification on sign in, unable to access email` and explain your issue.

**Reply by baharbaba** [reply]
i have been waiting for 5 days no response came from github support so i tried asking there

**Reply by davevad93** [reply]
I see @ozturkyusufziya , support can take a while to respond because they have a long list of open tickets to deal with. I suggest you be patient because as I said it can take a long time, I hope they get back to you as soon as possible to solve your problem.

**Reply by baharbaba** [reply]
oaky thanks for your response

**Reply by rsisto** [reply]
Hi! 
did you get an answer for this issue? My cofounder, a second owner of the org got the same issue, the solution he got was to unlink the email and create a new account.
Thanks!

**Comment by aakrigo** [comment]
My problem he is Microsoft is not login and me no idia
